app/views/unattended/provisioning_templates/provision/autoyast_sles_default.erb

```diff
@@ -257,7 +273,7 @@ rm /etc/resolv.conf
         <product_dir>/Module-Basesystem</product_dir>
         <product>sle-module-basesystem</product>
       </listentry>
-  <% if os_minor >= 1 && host_param('sle-module-basesystem-url') -%>
+  <% if os_minor >= 1 && os_minor < 4 && host_param('sle-module-basesystem-url') -%>
```

Comment:
This would also disable the integration of  media-url  (sle-module-base-system-url) (line 262). Is this was we want?

Reply:
This will disable the entire Module-Python2 entry, which is what we want as it is no longer provided.
The Module-Basesystem entry that is still required is above this line.

Reply:
you are right. thanks.